scripts/alltools.tcl

```diff
@@ -437,3 +448,24 @@ proc ordnumber {string} {
   }
   return $string
 }
+
+proc ucfirst {string} {
```

Comment:
isn't this what [string totitle] in Tcl is for? It also lowercases the rest

Reply:
It is, my bad...
I remove this useless proc